uaclient/entitlements/repo.py

```diff
@@ -124,6 +124,9 @@ def _perform_enable(self, silent: bool = False) -> bool:
 
     def _perform_disable(self, silent=False):
         if self.purge and self.origin:
+            print(messages.PURGE_EXPERIMENTAL)
```

Comment:
I think this is not relevant because purge should never be in APIs (unless it becomes so safe in the future right)
So self.purge will always be false on API calls.

Reply:
Fair enough, this path should never be reached through the API